examples/whip-whep/main.go

```diff
@@ -0,0 +1,132 @@
+// SPDX-FileCopyrightText: 2023 The Pion community <https://pion.ly>
+// SPDX-License-Identifier: MIT
+
+//go:build !js
+// +build !js
+
+// whip-whep demonstrates the WHIP/WHEP specifications to exhange SDP descritpions
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net"
+	"net/http"
+
+	"github.com/pion/webrtc/v4"
+)
+
+var (
+	videoTrack  *webrtc.TrackLocalStaticRTP
+	rtpListener *net.UDPConn
+	err         error
```

Comment:
Global error is not a good idea and it also raises the linter's wrath on line 30